i18n/es/docusaurus-plugin-content-docs/current/data/indexers/README.mdx

```diff
@@ -1,14 +1,92 @@
 ---
-title: Descripción general del indexador
+title: [Inglés; ayuda por favor] Descripción general del indexador
```

Comment:
not sure what's happening here lol

in general, manual changes to the `/i18n` directory aren't really necessary. i don't think the changes here will have any breaking affect, so it should be fine to keep them. but, you could also safely remove any changes to this directory if you want